airflow/hooks/hive_hooks.py

```diff
@@ -383,8 +384,13 @@ def load_file(
                 hql += "PARTITIONED BY ({pfields})\n"
             hql += "ROW FORMAT DELIMITED\n"
             hql += "FIELDS TERMINATED BY '{delimiter}'\n"
-            hql += "STORED AS textfile;"
+            hql += "STORED AS textfile\n"
+            if tblproperties:
```

Comment:
I'll recommend checking `if tblproperties is not None` isntead, and changing tblproperties above to None